config/locales/diaspora/en.yml

```diff
@@ -345,6 +345,10 @@ en:
     tutorials: "tutorials"
     tutorial: "tutorial"
     irc: "IRC"
+    closed_pod: "This pod is currently closed to new registrations. However, you can still join the diaspora* network by registering on %{wiki}. Because all pods are interconnected, you will have access to the same content there."
```

Comment:
@Flaburgan I've just noticed that there's a link to the wiki here where it tells you to register. Could you check that, please, as you've taken this PR over? It seems wrong to me. Should it be `%{poduptime}`?

Reply:
Yeah I need to check the wording and design, there are links to the wiki and official website which will break when the new one will be online. However I think that this PR will be ready before that so it's probably fine at the moment and should be changed when we will remove the in-app help.

Reply:
Oh OK, I've just looked more carefully and seen that the wiki link is to the choosing a pod article; I thought it was just to the wiki home page. So at least it is a 'correct' page while the wiki still exists. Ignore my question. :)